src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -1630,43 +1625,33 @@
         /**
          * Toggles the "Select All" checkbox in the Include or Exclude tab.
          * @param group - The group identifier ("Include" or "Exclude") for which the checkboxes should be toggled.
-         * @param keyCode - The key code used to select the box.
          */
-        $scope.toggleSelectAllCheckbox = (group, keyCode) => {
+        $scope.toggleSelectAllCheckbox = (group) => {
             $scope.assignListToGroup(group);
-
-            if (keyCode === 13) { //Enter key needs manual flip
-                $scope.pageSelected = !$scope.pageSelected;
-            }
-
+            //only called when checkbox is clicked on
+            //therefore needs to be flipped to reflect the function of a checkbox
```

Comment:
Be sure to format comments with a space after the `//`, Uppercase for the first word, and period at the end. Thanks.

Reply:
Thank you for catching this!